tests/ci/test-vm.sh

```diff
@@ -42,15 +89,25 @@ virsh start "${IMAGE_KEY}"
 greenprint "🛃 Checking for SSH is ready to go"
 for LOOP_COUNTER in $(seq 0 30); do
     RESULTS="$(wait_for_ssh_up $GUEST_ADDRESS)"
-    if [[ $RESULTS == 1 ]]; then
-        echo "SSH is ready now! 🥳"
+    if [[ "$RESULTS" == 1 ]]; then
         break
     fi
     sleep 10
 done
 
-# Check image installation result
-check_result
 
-greenprint "Here is the resulted VM: $LIBVIRT_IMAGE_PATH"
+# Tests
+greenprint "🛃 Tests 🛃"
+
+test_is_centos
+
+test_neptune_is_installed
+
+# Wait a bit for the X and the app to start
+sleep 30
+test_gnome_is_running
+
+# FIXME: Neptune app is starting bu crashing. Uncomment this tests when
```

Comment:
Can we link an issue where this was reported so we know when it come become re-enabled ? 

```suggestion
# FIXME: Neptune app is starting but crashing. Uncomment this tests when
```

Reply:
We talk about this offline, but the thing is that we don't have bug for this because it's third party application and we don't know exactly where to report this. And we still don't know the source of the problem :disappointed:

Reply:
I would say just log it as a bug in Jira in Base Enablement for now, and you could add VROOM-12345 (without linking the url) for some context so we remember where to look.

Reply:
:ok_hand:  Done :smile: